source/module_cell/klist.cpp

```diff
@@ -175,16 +175,16 @@ bool K_Vectors::read_kpoints(const std::string &fn)
 		ofs << "1 1 1 0 0 0" << std::endl;
 		ofs.close();
 	}
-    else if (GlobalV::KSPACING > 0.0)
+    else if (GlobalV::KSPACING[0] > 0.0)
```

Comment:
Should we make sure that GlobalV::KSPACING[1] > 0.0 and Global::KSPACING[2] > 0.0?